test/fixtures/tape-standalone.js

```diff
@@ -10,8 +10,12 @@ if (navigator.userAgent.includes("Chrome/")) {
 }
 
 // Export as a global a wrapped test function which enforces a timeout by default.
-window.test = (desc, fn) => {
-  tape(`${desc} (${browser})`, async (t) => {
+window.test = (desc, options, fn) => {
+  if (!fn && typeof options === "function") {
+    fn = options;
+    options = undefined;
+  }
+  tape(`${desc} (${browser})`, options, async (t) => {
     t.timeoutAfter(DEFAULT_TIMEOUT);
```

Comment:
You still need to remove the `timeoutAfter` call. Otherwise you cannot have a shorter timeout (there would be two timeouts).

So, declare an `options` object unconditionally with default value `{timeout: DEFAULT_TIMEOUT}`, and use `Object.assign` to merge the input `options` parameter.

And to avoid reshuffling of arguments, let's put `options` at the third place, after `fn`. Although `tape` doesn't document it, the parameters can be in any order: https://github.com/substack/tape/blob/v4.5.0/lib/test.js#L18-L37

Reply:
I’ve now implemented this using a&nbsp;function similar to&nbsp;the&nbsp;one used by&nbsp;`tape`.